commands/utils/validate.js

```diff
@@ -146,9 +144,16 @@ module.exports = validate_config = function (lt_config, validation_configs) {
           reject("Error!! Cypress Config File does not has correct json");
         }
       }
+    } else if (
+      semverCompare(cypress_version, "10.0.0") >= 0 &&
+      lt_config["run_settings"]["cypress_config_file"] &&
+      lt_config["run_settings"]["cypress_config_file"] != ""
+    ) {
+      reject(
+        'Error!! --ccf flag or cypress_config_file is not cupported with cypress>=10,use \n --cy="--config-file <file path>"'
```

Comment:
Should this be `--ccf flag and cypress_config_file` instead of or? As we are checking && in the above if statement...

Reply:
yes , nice catch